bigchaindb/pipelines/block.py

```diff
@@ -1,6 +1,6 @@
 """This module takes care of all the logic related to block creation.
```

Comment:
I think it is redundant. The directory/sub-package `pipelines` acts as a namespace, so I do not understand the need for re-specifying "pipeline".